src/FreqWindow.cpp

```diff
@@ -608,19 +610,30 @@ void FrequencyPlotDialog::GetAudio()
       }
       const auto nChannels = track->NChannels();
       if (track->GetRate() != mRate) {
-         AudacityMessageBox(
-            XO(
-"To plot the spectrum, all selected tracks must be the same sample rate.") );
+         using namespace BasicUI;
+         ShowMessageBox(
+            XO("To plot the spectrum, all selected tracks must be the same sample rate."),
```

Comment:
I think this other old message should be corrected to "must have"

Or maybe totally different words

@LWinterberg

Reply:
"must have" sounds good to me.

Reply:
Addressed.